recipes/liknorm-py/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "liknorm-py" %}
+{% set version = "1.0.3" %}
+{% set sha256 = "da072f7eba82741d47024d690b63ae85a9528f79e6b097210a2d64556907103a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pytest-runner
```

Comment:
`pytest-runner` is only build dependency if you add `pytest`, `test` or `ptr` as a command line arg to setup.py. Since none of those are in the `build/script:`, you can drop `pytest-runner`